src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -67,16 +68,13 @@ public LiveStripeResponseGetter(HttpClient httpClient) {
       raiseMalformedJsonError(responseBody, responseCode, requestId);
     }
 
-    if (resource instanceof StripeObject) {
-      StripeObject obj = (StripeObject) resource;
-      obj.setLastResponse(response);
-    }
+    resource.setLastResponse(response);
```

Comment:
Because of the type constraint, we no longer need to do an ugly check and cast at runtime here.